apps/design/backend/src/app.test.ts

```diff
@@ -365,147 +334,157 @@ test('Election package export', async () => {
   });
   const { election: appElection } = await apiClient.getElection({ electionId });
 
-  const electionPackageContents = await exportElectionPackage({
+  const electionPackageFilePath = await exportElectionPackage({
     apiClient,
     electionId,
     workspace,
   });
 
-  // Check overall structure of zip file
-  const zip = await JsZip.loadAsync(electionPackageContents);
-  expect(Object.keys(zip.files).sort()).toEqual([
-    'appStrings.json',
-    'election.json',
-    'systemSettings.json',
-    'vxElectionStrings.json',
-  ]);
-
-  //
-  // Check appStrings.json
-  //
-
-  const appStrings = safeParseJson(
-    await assertDefined(zip.file('appStrings.json')).async('text'),
-    UiStringsPackageSchema
+  const {
+    electionDefinition,
+    systemSettings,
+    uiStringAudioClips,
+    uiStringAudioIds,
+    uiStrings,
+  } = (
+    await readElectionPackageFromFile(electionPackageFilePath)
```

Comment:
Updated tests to use the `libs/backend` function for parsing election packages, since that's what we actually use in apps